src/molecules/formfields/CreditCardField/index.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Icon from 'atoms/Icon';
+
+import formStyles from 'molecules/formfields/shared/formfields.module.scss';
+import creditCardStyles from './credit_card_field.module.scss';
+
+const styles = Object.assign({}, formStyles, creditCardStyles);
+
+function CreditCardField(props) {
+  const {
+    className,
+    label,
+    input,
+    meta,
+    cardType,
+  } = props;
+
+  const baseClass = [
+    styles['formfield'],
+    styles['credit-card'],
+    { [styles.focused]: meta && meta.active },
+    className
+  ];
+
+  return (
+    <div className={classnames(...baseClass)}>
```

Comment:
this looks great, but I think another implementation would be to do this the way that John suggested we do the `DateField` - that is, make the RCL export a wrapper component that just organizes the components, and then the implementer would use `TextField` components for the three inputs (would probably need variants). This would allow us to wrap each input in a redux-form `Field` component and would prevent us from having to split out the values in the RCL